### ISSUE TITLE
Allow teams to advance progression in the casting table. (suggestion)

### ISSUE THREAD

**Issue by Eios29** [ReikaKalseki/Reika_Mods_Issues#1291]
This is not a bug per say, but it is quite an annoying feature.  If I am not mistaken casting table progression flags are bound to the owner of the table.  Now you can let others use your casting table with a key but, this does not allow them to advance the progression flag to my knowledge for "More Complex Crafting" or "Weak Magical Foci"  I bring this up as an issue due to playing with this mod on a private server and grouping up with another.  We started CrC together and would like to continue to do so without being forced to make several altars or completely rip a new altar apart reverting it to place my casting table on it for 0.2 seconds.  I understand the reasoning behind barring the progression this way but it is very frustrating as well.  Thank you.

**Comment by metnavman** [comment]
Opens up the issue of one person doing "all the work" and then someone else coming along and doing whatever they want. I don't speak for Reika, but I'd wager this goes against the spirit of the mod.

**Comment by AshIndigo** [comment]
If this was added then I would like to see a partial version of it.
Certain progressions would be marked "key progressions" which couldnt be shared. I.E: All colors, Fabrication, Rune crafting, Multiblock casting, Pylon casting and potentially some dimension progression.

**Comment by Eios29** [comment]
I don't believe all progression flags should be able to be advanced together.  Just the casting altar ones.  It's silly to have to build a completely new altar and craft every item again even though I've done it all just on someone else's table.  Make it a player flag rather then the table being the tracking device.  The player knows the knowledge right? A table should not be able to keep track of progress ;p anywho great mod just my opinion.  Do what you will.

**Comment by MineBartekSA** [comment]
In config there could be an option to enable a "Sharing progress" or "Altar flaging" option, so if you play with your friend on private server you can enable this. After all this is a good idea.
